renovate.json

```diff
@@ -10,6 +10,7 @@
     ],
     "commitMessagePrefix": "chore(all): ",
     "commitMessageAction": "update",
+    "labels": ["automerge"],
```

Comment:
Since you are touching this file would you mind adding this line so that dep updated are grouped: https://github.com/googleapis/google-cloud-go/blob/master/.github/renovate.json#L13

Reply:
You got it! Done.